verification/curator-service/ui/src/components/AppModal.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+    AppBar,
+    IconButton,
+    Modal,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import React, { ReactNode } from 'react';
+import { Theme, createStyles } from '@material-ui/core/styles';
+
+import CloseIcon from '@material-ui/icons/Close';
+import { WithStyles } from '@material-ui/core/styles/withStyles';
+import { withStyles } from '@material-ui/core';
+
+const styles = (theme: Theme) =>
+    createStyles({
+        modalContents: {
+            backgroundColor: theme.palette.background.paper,
+            left: '300px',
```

Comment:
Is this relatively safe cross-platform/resolution?

Reply:
Should be fine on all desktop/laptop. Mobile is a good question but I don't think we're targeting mobile yet. I assume we'll need a bit of a bugbash if we want to make everything work on mobile.